cmd/sharded-test-server/frontproxy.go

```diff
@@ -114,7 +116,7 @@ func startFrontProxy(
 
 	// create serving cert
 	hostnames := sets.NewString("localhost", hostIP)
-	klog.Infof("Creating kcp-front-proxy serving cert with hostnames %v", hostnames)
+	logger.Info("Creating kcp-front-proxy serving cert with hostnames", "hostnames", hostnames)
```

Comment:
nit: we're trying to use lower-case for messages

Reply:
Done :+1: